### PR TITLE
Expose method for gRPC connection termination

### PR DESCRIPTION
**Pull request by smacker**
Fix: #71

### CODE REVIEW

File: src/main/scala/org/bblfsh/client/BblfshClient.scala [hunk 1]
```diff
@@ -52,6 +52,10 @@ class BblfshClient(host: String, port: Int, maxMsgSize: Int) {
     stub.version(req)
   }
 
+  def shutdownNow(): Unit = {
```
Comment (smola) [inline src/main/scala/org/bblfsh/client/BblfshClient.scala]: maybe expose as the more standard `close()`?

Comment (smacker) [reply src/main/scala/org/bblfsh/client/BblfshClient.scala]: the name is taken from the issue description. I don't have any opinion about which name is better. Maybe maintainer @juanjux should decide?

Comment (juanjux) [reply src/main/scala/org/bblfsh/client/BblfshClient.scala]: I agree, close() is better (same as files sockets, etc)

**Comment by smacker** [comment]
renamed & amended & repushed

**Pull request merged**